config/locales/en.yml

```diff
@@ -31,3 +31,224 @@
 
 en:
   hello: "Hello world"
+
+  language:
+    en: "EN"
+    nb: "NO"
+
+  day:
+    monday: "Monday"
+    tuesday: "Tuesday"
+    wednesday: "Wednesday"
+    thursday: "Thursday"
+    friday: "Friday"
+    saturday: "Saturday"
+    sunday: "Sunday"
+
+  months:
+    january: "January"
+    february: "February"
+    march: "March"
+    april: "April"
+    may: "May"
+    june: "June"
+    july: "July"
+    august: "August"
+    september: "September"
+    october: "October"
+    november: "November"
+    december: "December"
```

Comment:
In Norway we have the calendar shows up as Monday, Tuesday, Wednesday, Thursday, Friday, Saturday, Sunday.
Instead of Sunday, Monday, Tuesday, Wednesday, Thursday, Friday, Saturday.

Reply:
You are correct about rails have built in locals for months and day names

This is the standard locals for day names in rails:
```
  date:
    day_names:
      - Sunday
      - Monday
      - Tuesday
      - Wednesday
      - Thursday
      - Friday
      - Saturday
```

This is what I wrote, changing the order:
```
  date:
    day_names:
      - Monday
      - Tuesday
      - Wednesday
      - Thursday
      - Friday
      - Saturday
      - Sunday
```